src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -68,6 +56,24 @@ MenuGame::MenuGame(Steam::steam_manager& mSteamManager,
                 downAction();
             }
         };
+    game.onEvent(Event::EventType::KeyReleased) +=
+        [this](const Event& mEvent)
+        {
+          if(!noActions) return;
+          if(!(--noActions)) clearDialogBox();
+        };
+    game.onEvent(Event::EventType::MouseButtonReleased) +=
+        [this](const Event& mEvent)
+        {
+			if(!noActions) return;
+            if(!(--noActions)) clearDialogBox();
+        };
+    game.onEvent(Event::EventType::JoystickButtonReleased) +=
+        [this](const Event& mEvent)
+        {
+          if(!noActions) return;
+          if(!(--noActions)) clearDialogBox();
+        };
```

Comment:
```suggestion
    const auto clearBoxIfActionNeeded = [this](const Event& mEvent)
    {
        if(!noActions) return;
        if(!(--noActions)) clearDialogBox();
    };

    game.onEvent(Event::EventType::KeyReleased) += clearBoxIfActionNeeded;
    game.onEvent(Event::EventType::MouseButtonReleased) += clearBoxIfActionNeeded; 
    game.onEvent(Event::EventType::JoystickButtonReleased) += clearBoxIfActionNeeded;
```

I think the above should work and help reduce repetition. Also, some comments would be nice in the body of the lambda expression, it is not immediately clear what `!(--noActions)` means (not syntactically, but in terms of what it means for the player)